cpg_infra/driver.py

```diff
@@ -2504,13 +2512,27 @@ def setup_container_read_write_permissions(self):
             self.infra.get_pulumi_name('full-in-images-writer-group-member'),
             self.full_group,
         )
+        self.images_dev_writer_group.add_member(
+            self.infra.get_pulumi_name('dev-images-writer-group-member'),
```

Comment:
Is this right? This pulumi is clear as mud to me, but other similar things seem to follow the pattern `'dev-in-images-writer-group-member'`.

Reply:
Good point @jmarshall, by reviewing existing groups I can see there are no actual 'dev' groups, there are only 'test-full' and 'test'. I had adjusted the get_pulumi_name calls.